new theme,added factorial and percentage button, fixed centering

FilipposFC opened the pull request.

Emilance commented:
Great Job 💪, I appreciate your contribution @FilipposFC  👏👏👏 ,  
 the calculator's width look  smaller than it was, kindly   return the operators' color  to  what  it was (I mean the same color with other buttons) and adjust the calculator's width  thereafter I will accept the pull request

FilipposFC commented:
Thanks for answering so quickly!!
 I changed the calculator width to what it was before. I also change the color of the operator buttons to what it was.
Moreover i added the power button to calculator in addition to the percentage and factorial buttons I added before. Lastly I made a new ccs file for the dotBubbles theme that i created so that the code is more neat and clean.

Emilance commented:
Awesome @FilipposFC ,  Thanks for your contribution 👏👏, I will merge now
